source/_docs/automation/editor.markdown

```diff
@@ -31,3 +31,7 @@ message: Sensor value greater than 10
 
 Automation created or edited via the user interface, are activated immediately
 after save the automation.
+
+## Split Config
+
+When creating automations using the GUI, they are written to `automations.yaml` even if you have a [split config](/docs/configuration/splitting_configuration/#example-combine-include_dir_merge_list-with-automationsyaml). Make sure you are correctly including the base `automations.yaml` file.
```

Comment:
This is already explained here: <https://www.home-assistant.io/docs/configuration/splitting_configuration/#example-combine-include_dir_merge_list-with-automationsyaml>

Should we link to that for context?

Additionally:
- Please avoid the term `config`. It is a tech abbreviation, use `Configuration` instead. See also: <https://developers.google.com/style/word-list#config>
- It could use some more introduction. It feels kinda "out-of-the-blue" on the bottom of this page.